docs/source/getting-started-client-mode.md

```diff
@@ -0,0 +1,98 @@
+## Enabling Client Mode/Standalone Support
+
+Even if you are not leveraging the full distributed capabilities of Jupyter Enterprise Gateway, the client mode can
+still help mitigate the resource estarvation by enabling pseudo-distributed mode where kernels are started in client
+mode in different nodes of the cluster utilizing a round-robin algorithm. In this case, you can still get bottlenecks
+in a given node that might get requests to start "large" kernels, but otherwise, you will be better of compared to when
+all kernels are stsarted from a single node or as local processes which is the default for vanilla Jupyter Notebook.
+
+The pseudo-distributed capabilities are currently supported in YARN Client mode or using vanilla Spark Standalone and 
+require the following environment variables to be set:
+
+* SPARK_HOME: Must point to the Apache Spark installation path
+```
+SPARK_HOME:/usr/hdp/current/spark2-client                            #For HDP distribution
+```
+
+**Configuring Kernels for YARN Client mode**
+
+For each supported Jupyter Kernel, we have provided sample kernel configuration and launcher available as part of the release
```

Comment:
Suggested change:
...`we have provided sample kernel configurations and launchers as part`...